src/gui/mainwindow.cpp

```diff
@@ -2061,31 +2044,12 @@ void MainWindow::checkProgramUpdate()
 #endif
 
 #ifdef Q_OS_WIN
-bool MainWindow::addPythonPathToEnv()
-{
-    if (m_hasPython) return true;
-
-    QString pythonPath = Preferences::getPythonPath();
-    if (!pythonPath.isEmpty()) {
-        Logger::instance()->addMessage(tr("Python found in '%1'").arg(Utils::Fs::toNativePath(pythonPath)), Log::INFO);
-        // Add it to PATH envvar
-        QString pathEnvar = QString::fromLocal8Bit(qgetenv("PATH").constData());
-        if (pathEnvar.isNull())
-            pathEnvar = "";
-        pathEnvar = pythonPath + ';' + pathEnvar;
-        qDebug("New PATH envvar is: %s", qUtf8Printable(pathEnvar));
-        qputenv("PATH", Utils::Fs::toNativePath(pathEnvar).toLocal8Bit());
-        return true;
-    }
-    return false;
-}
-
 void MainWindow::installPython()
 {
     setCursor(QCursor(Qt::WaitCursor));
     // Download python
     const QString installerURL = ((QSysInfo::windowsVersion() >= QSysInfo::WV_VISTA)
-                                  ? "https://www.python.org/ftp/python/3.5.2/python-3.5.2.exe"
+                                  ? "https://www.python.org/ftp/python/3.6.6/python-3.6.6.exe"
```

Comment:
Search plugins seems to work with 3.7.

Reply:
I ran into problems with pip with py 3.7 on windows, so I guess it wasn't stable enough.